flows/codelingo/review/review/cli.go

```diff
@@ -222,3 +276,23 @@ func reviewAction(cliCtx *cli.Context) (chan proto.Message, <-chan *flowutil.Use
 	resultc, userVarc, errc, err := RequestReview(ctx, req, insecure)
 	return resultc, userVarc, errc, cancel, errors.Trace(err)
 }
+
+func isGoProject(currentDir string) (bool, error) {
+	isGo := false
+	items, err := ioutil.ReadDir(currentDir)
+	if err != nil {
+		return false, err
+	}
+	for _, item := range items {
```

Comment:
```suggestion
	for _, item := range items {
	if !item.IsDir() && filepath.Ext(item.Name()) == ".go" {
	    return true, nil
	}
```
If you use the recursive method you should exit before recursion, otherwise we'll have to check every single file.